pytorch_lightning/strategies/ddp.py

```diff
@@ -424,8 +424,8 @@ def post_training_step(self):
             self.model.require_backward_grad_sync = True
 
     @classmethod
-    def register_plugins(cls, plugin_registry: Dict) -> None:
-        plugin_registry.register(
+    def register_strategies(cls, strategies_registry: Dict) -> None:
+        strategies_registry.register(
```

Comment:
Shouldnt it be "strategy registry"? Not plural

Reply:
Updated!